tensorflow/stream_executor/dnn.cc

```diff
@@ -122,7 +122,11 @@ string PadAlignmentString(PadAlignment alignment) {
       return "cuDNN padding";
     case PadAlignment::kTensorFlowPadding:
       return "TensorFlow padding";
+    default:
+      LOG(FATAL) << "Unknown padding alignment "
```

Comment:
Good point. I was trying to follow the style in the file. For example: in the methods 'FilterLayoutString' and 'ShortPoolingModeString', the default case is LOG(FATAL).